preCICE-Ubuntu-20.04/install-precice.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+set -ex
+
+USER="vagrant"
+
+# Download the preCICE Ubuntu 20.04 package from GitHub, install it, and cleanup
```

Comment:
Maybe using a debug build from source would be more helpful here because it allows to output debug log.
Nobody will notice the speed drop in the vm :wink:

Reply:
Right... I always forget that our packages do not support Debug mode...

Reply:
Replaced this with preCICE built in Debug mode (build a Debian package and install it).